src/concrete/ml/sklearn/tree_to_numpy.py

```diff
@@ -141,12 +146,19 @@ def add_transpose_after_last_node(onnx_model: onnx.ModelProto):
     # Get the output node
     output_node = onnx_model.graph.output[0]
 
-    # Create the node with perm attribute equal to (2, 1, 0)
+    # When using FHE sum for tree ensembles, create the node with perm attribute equal to (1, 0)
```

Comment:
could you maybe add a comment explaining why this change is needed (and what is this perm attribute) ?

Reply:
done